monero-core.pro

```diff
@@ -86,7 +86,8 @@ linux {
         -lssl \
         -lcrypto \
         -Wl,-Bdynamic \
-        -lunwind \
+        # currently monero has an issue with "static" build and linunwind-dev
+        # -lunwind \
```

Comment:
Only for one of the distros, though... I think this will break the Arch package. That was the reason for 010a215. Hm.. ok, how about we add another CONFIG switchs for `libunwind_on` and `libunwind_off`, turn it on by default, so so near the top
```
CONFIG += libunwind_on
```
and then change the above lines to
```
CONFIG(libunwind_off) {
   message(Building without libunwind)
} else {
    message(Building with libunwind)
    LIBS += -Wl,-Bdynamic -lunwind
}
```
It might read a bit oddly with two flags, only because I'm not sure how to do it with one flag. It would need to be set by default, but we must also be able to remove it via command line -- is that possible? (maybe somehow, like `CONFIG -= libunwind_on`). In any case the above should do the job.

Then on Ubuntu, update readme instructions to pass `libunwind_off`when building on Ubuntu -- until this static linking problem is fixed. Distros that are not broken will continue to work by default without any extra flags, as desired.